docs/niscope/class.rst

```diff
@@ -2341,7 +2343,7 @@ cable_sense_mode
 
     .. py:attribute:: cable_sense_mode
 
-        Specifies whether and how the oscilloscope is configured to generate a CableSense signal on the specified channels when the :py:meth:`niscope.Session.cable_sense_signal_start` method is called.
+        Specifies whether and how the oscilloscope is configured to generate a CableSense signal on the specified channels when the :py:meth:`niscope.Session.CableSenseSignalStart` method is called.
```

Comment:
This is odd. `cable_sense_mode` is a "Cable sense" feature property. Why is that in the API but `CableSenseSignalStart` is not? Seems like a bug.

Reply:
#1256